Makefile

```diff
@@ -50,12 +55,12 @@ prepare_seed_data:
 
 load_demo_data: ## Import example dataset
 load_demo_data: FLAGS := $(if $(FLAGS),$(FLAGS),--config=config.yaml)
-load_demo_data: | dependencies prepare_seed_data
+load_demo_data: | dependencies_no_js prepare_seed_data
```

Comment:
we don't need them to load demo data in the backend. In the Dockerfile we drop the `node_modules` after building the bundle, but then if we run `make load_demo_data` to create the tables and such (as instructed), it takes forever and uses a lot of disk space because it recreates all of the js dependencies

Reply:
Oooh I see! Yes, makes sense.